src/tox_lsr/test_scripts/runwoke.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Do not exit on an error to continue ansible-doc and ansible-test.
+set -euo pipefail
+
+#uncomment if you use $ME - otherwise set in utils.sh
+#ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "${SCRIPTDIR}/utils.sh"
+
+curl -sSfL https://git.io/getwoke | bash -s -- -b "$LSR_TOX_ENV_DIR/bin"
+WOKE_CMD="$LSR_TOX_ENV_DIR/bin/woke"
+WOKE_OUT="$LSR_TOX_ENV_DIR/tmp/woke_output.txt"
+
+trap "rm -f $TOXINIDIR/.woke.yml" RETURN
+
+pushd "$TOXINIDIR" > /dev/null
+cp "$LSR_CONFIGDIR/woke.yml" .woke.yml
+"$WOKE_CMD" > "$WOKE_OUT" 2>&1
```

Comment:
can't you use `"$WOKE_CMD" -c "$LSR_CONFIGDIR/woke.yml"` ?

Reply:
Also, with the `set -e` above - if `"$WOKE_CMD"` exits with a non-zero exit code, won't the script exit there?  I think you'll need to use `"$WOKE_CMD" -c "$LSR_CONFIGDIR/woke.yml" > "$WOKE_OUT" 2>&1 || :` to ignore any error return from woke